src/PKSim.Core/Snapshots/Mappers/SimulationMapper.cs

```diff
@@ -142,9 +147,33 @@ private string[] usedObervedDataFrom(ModelSimulation simulation, PKSimProject pr
 
          updateUsedObservedData(simulation, snapshot.ObservedData, project);
 
+         await runSimulation(snapshot, simulation);
+
+         simulation.AddAnalyses(await simulationChartFrom(simulation, snapshot.Charts, project));
          return simulation;
       }
 
+      private async Task runSimulation(SnapshotSimulation snapshot, ModelSimulation simulation)
+      {
+         if (!snapshot.HasResults)
+            return;
+
+         await _simulationRunner.RunSimulation(simulation, createDefaultAnalysis: false);
```

Comment:
we are running simulations during the snapshot load! in case of long pop simulation, time to get some coffee